src/irc/core/irc.c

```diff
@@ -281,6 +284,31 @@ char *event_get_params(const char *data, int count, ...)
 	return duprec;
 }
 
+/* Given a string containing <params>, strip any colon prefixing <trailing>. */
+static void strip_params_colon(char *const params)
+{
+	if (!params) {
```

Comment:
now we just need to get your code into the same "style" as irssi. instead of !var we have to use `var ==  NULL`

Reply:
Ah yeah. I will update all these. Thanks! Having clang-format will be nice!